packages/generator-react-server/generators/app/templates/_reactserverrc

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I guess the only down-side of having all of the config defaults listed here is we need to make sure it stays in sync with the actual defaults. 😛